x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_list.tsx

```diff
@@ -89,6 +89,12 @@ const TRUSTED_APPS_PAGE_LABELS: ArtifactListPageProps['labels'] = {
       defaultMessage: '"{itemName}" has been removed from trusted applications.',
       values: { itemName },
     }),
+  emptyStateTitleNoEntries: i18n.translate(
+    'xpack.securitySolution.trustedApps.emptyStateTitleNoEntries',
+    {
+      defaultMessage: 'There are no trusted application entries to display.',
```

Comment:
Should this be `There are no trusted applications to display.` instead?

Reply:
🦅 👁️ ! d080743